src/scripts/scenes/xr-scene.js

```diff
@@ -298,217 +270,6 @@ export default class XrScene {
     return this.frame;
   };
 
-  /**
-   * For every frame this function updates the controller/laser transforms
-   * and raycast intersections from the controller laser.
-   * Keeps track of what input sources are currently in the scene and
-   * handles meshes accordingly when sources are added and removed
-   * @param {XRFrame} xrFrame
-   * @param {XRReferenceSpace} xrRefSpace
-   */
-  _updateInputSources(xrFrame, xrRefSpace) {
-    const inputSources = XR.session.getInputSources();
-
-    // For the number of input sources in the current session
-    for (let i = 0; i < inputSources.length; i++) {
-      const inputSource = inputSources[i];
-
-      if (inputSource) {
-        // Should the input source support visual laser raycasting?
-        const isTrackedPointer = inputSource.targetRayMode === 'tracked-pointer';
-
-        // If can handle visual lasers and has a grip matrix indicating that
-        // the controller is a visual element in the immersive scene
-        if (isTrackedPointer && inputSource.gripSpace) {
-          // Get grip space pose for controller
-          const gripPose = xrFrame.getPose(inputSource.gripSpace, xrRefSpace);
-
```

Comment:
Controller rendering is no longer handled in your changes. This needs to be re-introduced before merging as there is currently no way to see the controller or its laser.